desktop_version/src/Music.cpp

```diff
@@ -260,6 +262,15 @@ void musicclass::play(int t)
 				// musicchannel = musicchan[currentsong].play(0);
 				// musicchannel.soundTransform = new SoundTransform(0);
 				// musicchannel.addEventListener(Event.SOUND_COMPLETE, loopmusic);
+				if (Mix_FadingMusic() == MIX_FADING_OUT) {
+					// We're already fading out
+					fadeoutqueuesong = t;
+					currentsong = -1;
+					if (!dontquickfade)
+						Mix_FadeOutMusic(500); // fade out quicker
+					else
+						dontquickfade = false;
+				} else
```

Comment:
else on separate line

Reply:
Fixed in cd31f0dbab85da2d38050231866a1b62662a7108.